src/AuthMail.php

```diff
@@ -57,7 +57,7 @@ public static function getSectorizedDetails(): array
 
     public function prepareInputForUpdate($input)
     {
-        if (empty($input['name'])) {
+        if (isset($input['name']) && empty($input['name'])) {
```

Comment:
`isset($input['name'])` will be false if the passed value is `null`, but in that case, the name field value will be erased by an empty value and we do not want this to happen.

```suggestion
        if (array_key_exists('name', $input['name']) && empty($input['name'])) {
```